packages/store-sync/src/postgres-decoded/buildColumn.ts

```diff
@@ -14,12 +14,12 @@ export function buildColumn(name: string, schemaAbiType: SchemaAbiType) {
     case "int8":
     case "int16":
     case "uint24":
-    case "uint32":
```

Comment:
`int32` as well?

Reply:
No I don't think so, `int32` covers the exact same range as postgres `integer` - they are both 32 byte signed integers from -2147483648 to +2147483647.

Reply:
should we make em the same type for consistency? feels weird if `uint32` is a `bigint` but `int32` is a `integer`?

Reply:
I agree it's not obvious and requires thinking a bit about Solidity AND Postgres. I'm going just move it, I like the symmetry as well.